rudder-lang/src/ast/resource.rs

```diff
@@ -33,7 +33,8 @@ use super::enums::*;
 use super::value::Value;
 use crate::error::*;
 use crate::parser::*;
-use std::collections::{HashMap, HashSet};
+use std::collections::{HashMap, HashSet, hash_map::Entry};
+// use std::collections::hash_map::Entry;
```

Comment:
no need to keep this line (as it is now part of the other `use` line)

Reply:
yes, my bad. Wasn't sure about the cleanest way to include it